src/components/FlowList.vue

```diff
@@ -1,35 +1,59 @@
 <template>
   <div class="flow-list">
-    <p-layout-table sticky>
+    <p-layout-table :sticky="media.lg">
       <template #header>
-        <div class="flow-list__header">
-          <div class="flow-list__header-start">
-            <template v-if="selected.length == 0">
-              <span v-if="!!flowsCount && !!deploymentsCount" class="flow-list__results-count">
-                <ResultsCount :label="localization.info.flow" :count="flowsCount" />
-                {{ localization.info.with }}
-                <ResultsCount :label="localization.info.deployment" :count="deploymentsCount" />
-              </span>
-            </template>
-
-            <template v-else-if="selected.length">
-              <SelectedCount :count="selected.length" />
-              <FlowsDeleteButton size="xs" :selected="selected" @delete="deleteFlows" />
-            </template>
-          </div>
+        <div class="flow-list__header-container">
+          <div class="flow-list__header">
+            <div class="flow-list__header-start">
+              <template v-if="selected.length == 0">
+                <span v-if="!!flowsCount && !!deploymentsCount" class="flow-list__results-count">
+                  <ResultsCount :label="localization.info.flow" :count="flowsCount" />
+                  {{ localization.info.with }}
+                  <ResultsCount :label="localization.info.deployment" :count="deploymentsCount" />
+                </span>
+              </template>
 
-          <div class="flow-list__header-end">
-            <SearchInput v-model="search" :placeholder="localization.info.searchByFlowName" :label="localization.info.searchByFlowName" />
-            <p-select v-model="filter.sort" :options="flowSortOptions" />
-            <p-tags-input v-model="filter.flowRuns.tags.name" :placeholder="localization.info.addTagPlaceholder" class="flow-list__flow-run-tags">
-              <template #empty-message>
-                <span class="flow-list__flow-run-tags--empty">{{ localization.info.filterByFlowRunTags }}</span>
+              <template v-else-if="selected.length">
+                <SelectedCount :count="selected.length" />
+                <FlowsDeleteButton size="xs" :selected="selected" @delete="deleteFlows" />
               </template>
```

Comment:
I'd like to see this piece abstracted into its own component. Decent bit of logic/template that could be separated out.

Reply:
Agreed - I don't have any (good) ideas for a more generic component but I could certainly split out the logic for this instance